.circleci/config.yml

```diff
@@ -91,7 +91,15 @@ jobs:
     steps:
       - checkout
       - run_tests:
-          test_type: unit
+          test_type: test-unit
+
+  lint:
+    docker:
+      - image: circleci/python:3.6
```

Comment:
can you add `environment: *global_environment_vars` to this job to be consistent with the rest of the jobs? I don't think this job will actually _need_ any of the current env vars, but the idea of that anchor was to provide a place to set global vars for every job.

Reply:
@alfredodeza did your force push overwrite this fix?

Reply:
Not sure... just pushed again to fix this